### ISSUE TITLE
how to fix BUILD FAILED

### ISSUE THREAD

**Issue by loogchin** [luckydog7/Funkin-android#1816]
Warning: Mapping new ns http://schemas.android.com/repository/android/common/02 to old ns http://schemas.android.com/repository/android/common/01
Warning: Mapping new ns http://schemas.android.com/repository/android/generic/02 to old ns http://schemas.android.com/repository/android/generic/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/addon2/02 to old ns http://schemas.android.com/sdk/android/repo/addon2/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/repository2/02 to old ns http://schemas.android.com/sdk/android/repo/repository2/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/sys-img2/02 to old ns http://schemas.android.com/sdk/android/repo/sys-img2/01
Warning: Mapping new ns http://schemas.android.com/repository/android/common/02 to old ns http://schemas.android.com/repository/android/common/01
Warning: Mapping new ns http://schemas.android.com/repository/android/generic/02 to old ns http://schemas.android.com/repository/android/generic/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/addon2/02 to old ns http://schemas.android.com/sdk/android/repo/addon2/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/repository2/02 to old ns http://schemas.android.com/sdk/android/repo/repository2/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/sys-img2/02 to old ns http://schemas.android.com/sdk/android/repo/sys-img2/01

Task :app:compileDebugJavaWithJavac
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: C:\Users\Administrator\Desktop\kade engine\Funkin-android-0.2.7.1.5\export\release\android\bin\app\src\main\java\org\libsdl\app\SDL.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.

Task :app:packageDebug FAILED

FAILURE: Build failed with an exception.

What went wrong:
Execution failed for task ':app:packageDebug'.
A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
com.google.common.base.VerifyException (no error message)

Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.6.3/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 32s
52 actionable tasks: 41 executed, 11 up-to-date

how to `fix
![จับภาพ](https://user-images.githubusercontent.com/87599145/130487625-79646a37-d2c4-4201-b07b-24761d398496.PNG)

**Comment by mozesnjoroge** [comment]
@loogchin. Your screenshot does not show any command I can use on the terminal to resolve the issue. Kindly assist.

**Comment by mkssanudeep** [comment]
Did you find any solution for the above?

**Comment by mozesnjoroge** [comment]
Yes I did. Thanks for asking. I deleted the Android SDK platforms folder via the terminal in Android Studio, then set about to re-install the same SDK platforms. In the process it cleared the error in the cache and my flutter apps build without hiccups. Oh, perform a gradle clean build using the command "/.gradlew clean build" or just "gradle clean build".

**Comment by mozesnjoroge** [comment]
Try these links from stack overflow: 
https://stackoverflow.com/questions/68789848/warning-mapping-new-ns-to-old-ns-and-emulator-stopping-abruptly
https://stackoverflow.com/questions/67645844/mapping-new-ns-to-old-ns

**Comment by mkssanudeep** [comment]
> 
> 
> Yes I did. Thanks for asking. I deleted the Android SDK platforms folder via the terminal in Android Studio, then set about to re-install the same SDK platforms. In the process it cleared the error in the cache and my flutter apps build without hiccups. Oh, perform a gradle clean build using the command "/.gradlew clean build" or just "gradle clean build".

Can I do it on command prompt?

**Comment by mozesnjoroge** [comment]
Sure you can
